hooks/conan-center.py

```diff
@@ -609,6 +610,12 @@ def test(out):
                 out.error("Both '{0}requires' attribute and '{0}requirements()' method should not "
                           "be declared at same recipe.".format(prefix))
 
+    @run_test("KB-H057", output)
+    def test(out):
+        class_name = type(conanfile).__name__
+        if class_name in ("LibnameConan", "ConanRecipe", "ConanFileDefault"):
```

Comment:
Let's allow `ConanRecipe` just in case we want to implement something around a common class name in the future of Conan. I think that should be a good common class name.
```suggestion
        if class_name in ("LibnameConan", "ConanFileDefault"):
```

Reply:
Okay, it would be nice to have `ConanRecipe` as a transitional name, merging.

Regarding the list of recipes - the hook is pre-export so it will affect only changed or rebuilt recipes.